spec/routemaster/integration/api_client_spec.rb

```diff
@@ -165,70 +156,58 @@ class ProcHandler
     end
   end
 
-  describe 'Future GET request' do
-    let(:body_cache_keys) { ["cache:#{url}", "v:,l:,body"] }
-    let(:headers_cache_keys) { ["cache:#{url}", "v:,l:,headers"] }
-    let(:url) { "#{host}/success" }
 
-    context 'when there is a previous cached resource' do
-      let(:cache) { Routemaster::Config.cache_redis }
+  describe 'caching behaviour' do
+    let(:url) { "#{host}/resources/1" }
+    def timestamp ; subject.get(url).body.updated_at ; end
 
-      it 'returns the response from within the future' do
-        expect(cache.hget("cache:#{url}", "v:,l:,body")).to be_nil
+    describe 'GET requests' do
+      context 'when the resource was fetched' do
+        let!(:cached_stamp) { timestamp }
+        let(:fetched_stamp) { timestamp }
```

Comment:
Maybe I'm confused but why the `fetched_timestamp` shouldn't just `timestamp` do for tests below, correct me if I'm wrong, but as it reads I'm a little confused why this is needed, forgive my Rspec 😸